### ISSUE TITLE
Assignment 1 R7.1

### ISSUE THREAD

**Issue by THardy98** [CISC-CMPE-327/Information#19]
"Logout will invalid the current session redirect to the login page. After logout, the user shouldn’t be able to access restricted pages."

For the first sentence:
"Logout will invalid the current session redirect to the login page."

is this intended to mean:
"Logout will invalid the current session **and** redirect to the login page."

as they are different things.

Also, are restricted pages anything except for the login page?

**Comment by steven-hh-ding** [comment]
Yes! You are right. If you don't mind, could you fork and create a pull request?

**Comment by steven-hh-ding** [comment]
restricted pages are defined based on their own requirements. i believe anything other than login & register

**Comment by THardy98** [comment]
Yup I can.

Quick question:

R1.9:
"For any formatting errors, render the login page and show the message 'email/password format is incorrect.'"

What is considered a formatting error for emails and passwords? (considering that there are already specifications for non-empty inputs, passwords of a certain length with upper/lower case and special character, and emails that follow the addr-spec)

**Comment by THardy98** [comment]
https://github.com/CISC-CMPE-327/Information/pull/20
^ pull request

**Comment by steven-hh-ding** [comment]
Thanks! The formatting errors refer the constraints above (what happens for the negative cases)

**Comment by THardy98** [comment]
I'm still not clear what the formatting constraints would be? 

The specifications I mentioned earlier were covered by other test cases.

**Comment by steven-hh-ding** [comment]
so this one is like what happed if you failed any of the previous constraints (e.g. password has only lower-case letters) and what message should be printed out

**Comment by THardy98** [comment]
But isn't that covered in A1 R1.8:

"Password has to meet the required complexity: minimum length 6, at least one upper case, at least one lower case, and at least one special character"

where we test all those constraints anyway?
